modules/es-extensions/publisher/extensions/app/greg-diff/themes/default/js/diff.js

```diff
@@ -51,12 +51,12 @@ $('.container-fluid').resize(function(){
  */
 function setViewPanelsHeight(){
     if($(window).width() > 768){
-        $(viewPanel).css('height', ($('body').height() - appBarHeight));
+        $(viewPanel).css('height', ($('body').height() - ($(window).height()*.1)));
```

Comment:
Here we are setting the height of code-mirror view panel height to 90% of the view port height relatively other than hard coding.